overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/rhcos-fips.sh

```diff
@@ -75,6 +84,8 @@ firstboot() {
               echo $(grep $line $f) >> $tmpfile
           done
       done
+      # we reboot before ignition runs, so let's keep this opt
```

Comment:
without this after reboot `/proc/cmdline` contains only `$ignition_firstboot`, so all ignition steps are skipped